app/controllers/activities_controller.rb

```diff
@@ -0,0 +1,36 @@
+class ActivitiesController < ApplicationController
+	include ActivitiesHelper
```

Comment:
Oh! thanks, I never knew of that
will do that ASAP

On Wed, May 20, 2015 at 4:10 PM, B V Satyaram <notifications@github.com>
wrote:

> In app/controllers/activities_controller.rb
> <https://github.com/chi6rag/HourGlass/pull/1#discussion_r30689025>:
>
> > @@ -0,0 +1,36 @@
> > +class ActivitiesController < ApplicationController
> > +	include ActivitiesHelper
>
> Now I understand the problem. I guess you haven't enabled convert tabs to
> spaces in sublime settings.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chi6rag/HourGlass/pull/1/files#r30689025>.
>